oak_loader/src/options.rs

```diff
@@ -223,25 +223,26 @@ fn create_sign_table(opt: &Opt) -> anyhow::Result<SignatureTable> {
         debug!("Parsed signature manifest file: {:?}", signatures_manifest);
 
         for (module_hash, signature_vec) in loaded_signatures_manifest.signatures.iter() {
-            let mut parsed_signatures = vec![];
-
-            for signature_item in signature_vec.iter() {
+            let mut loaded_signatures = vec![];
+            for signature_location in signature_vec.iter() {
                 debug!("Loading signature for {}", module_hash);
-                let public_key = PublicKey::from_file(&signature_item.public_key)
-                    .context("Couldn't parse public key file")?;
-                debug!("Parsed public key: {}", public_key.to_base64());
-                let signature = SignatureBox::from_file(&signature_item.signature)
-                    .context("Couldn't parse signature file")?;
-                debug!("Parsed signature: {}", signature.to_string());
-                parsed_signatures.push(Signature {
-                    public_key,
-                    signature,
-                });
+                let loaded_signature = match &signature_location {
+                    SignatureLocation::Path(path) => {
+                        let signature_file = read(&path)
```

Comment:
I think in a case of a corrupted file it's better for the maintainer (a person who runs Oak) to just exclude it from the `signatures.toml`, because it's a critical error and it might be easy to miss it as an `error!` log.
And if a signature couldn't be loaded, Oak will work fine, but clients wouldn't be able to connect to Oak - and it's harder to check.